### ISSUE TITLE
Fix api/web subdomain configuration

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/ocl_issues#102]
It appears that the api vs. web subdomains are configured incorrectly that in some cases is exposing private data to unauthenticated users. This appears to be happening on all servers. See several examples where a request for web content instead results in JSON content:

This is not a valid web request but it returns the JSON from the equivalent API request. This request requires authentication from the API, but it allows an anonymous user to get results:
```
https://qa.openconceptlab.org/users/
https://api.qa.openconceptlab.org/users/
```

Same issue with orgs:
```
https://qa.openconceptlab.org/orgs/
https://api.qa.openconceptlab.org/orgs/
```

Same results on other servers:
```
https://openconceptlab.org/orgs/
https://demo.openconceptlab.org/orgs/
https://staging.openconceptlab.org/orgs/
```

On QA, there seem to be some additional issues (maybe these haven't been promoted to other servers yet). Viewing the list of sources and collections for a user displays JSON results:
```
https://qa.openconceptlab.org/users/admin/sources/
https://qa.openconceptlab.org/users/admin/collections/
```

Attempting to view this source and collection in OCL-WEB results in an error:
```
https://qa.openconceptlab.org/users/admin/sources/b5f61cfb-4ee6-4c2a-99b7-87fdcfd6f56b/
https://qa.openconceptlab.org/users/admin/collections/b5f61cfb-4ee6-4c2a-99b7-87fdcfd6f56b/
```

Accessing these repos from the API works fine:
```
https://api.qa.openconceptlab.org/users/admin/collections/b5f61cfb-4ee6-4c2a-99b7-87fdcfd6f56b/
https://api.qa.openconceptlab.org/users/admin/sources/b5f61cfb-4ee6-4c2a-99b7-87fdcfd6f56b/
```

**Comment by paynejd** [comment]
Flagging this as high priority

**Comment by rkorytkowski** [comment]
So it's interesting. It seems that currently the web layer is either using an admin token or an anonymous token to run REST queries against API. Not sure what exactly is the anonymous token, because it is set to the value of "root" token in environment variables and I do not see such a user in API. I can create a regular user for the anonymous token to address this issue, but still the web layer authentication is broken conceptually. Even though I'm logged in as "rafal" all my actions are seen as done by the "root". It's best we address #40 ASAP.

**Comment by rkorytkowski** [comment]
@paynejd, I've fixed this issue by securing the rest endpoints exposed by the web service to restrict access as follows:
/users -> admins
/users/(user)/sources -> admins and (user)
/users/(user)/collections -> admins and (user)
/orgs -> authenticated users

These endpoints are explicitly exposed by the web service. I'm guessing they are used by some ajax requests in UI, but I'm not sure where.

**Comment by paynejd** [comment]
It appears that all operations by authenticated users in OCL_WEB are performed by the root account. For example, on the demo server, I created a new account, verified the email address, created a new organization (via the web app), and the org audit info shows that it was created by root. I then used the API to login to get my token and created an org via the API, and it correctly showed that it was created by my new user account (and not by root).
